pretyping/evarconv.ml

```diff
@@ -568,6 +568,38 @@ let infer_conv_noticing_evars ~pb ~ts env sigma t1 t2 =
     if !has_evar then None
     else Some (UnifFailure (sigma, UnifUnivInconsistency e))
 
+let pr_econstr = ref (fun _ _ _ -> Pp.str "unable to print econstr")
```

Comment:
you don't need this, there's already the same thing at Termops.Internal

Reply:
Excellent, thank you.